doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -222,16 +228,33 @@ For devices that do not support :ref:`partition_manager`, you must declare the `
 Currently, the :ref:`zephyr:nrf54h20dk_nrf54h20` is the only device that requires manual partition definition.
 To see how to do this, refer to the example in the :file:`samples/bluetooth/fast_pair/input_device/boards/nrf54h20dk_nrf54h20_cpuapp.overlay` file.
 
-To build an application with the Fast Pair support, include the following additional CMake options:
+To generate a hexadecimal file with the Fast Pair provisioning data during the build run, you must use the default |NCS| build system configuration with sysbuild and set the following sysbuild Kconfig options:
 
-* ``FP_MODEL_ID`` - Fast Pair Model ID in format ``0xXXXXXX``,
-* ``FP_ANTI_SPOOFING_KEY`` - base64-encoded Fast Pair Anti-Spoofing Private Key.
+* ``SB_CONFIG_BT_FAST_PAIR_MODEL_ID`` - Fast Pair Model ID in format ``0xXXXXXX``,
+* ``SB_CONFIG_BT_FAST_PAIR_ANTI_SPOOFING_PRIVATE_KEY`` - base64-encoded Fast Pair Anti-Spoofing Private Key.
 
 The Fast Pair partition address is provided automatically by the build system.
 
-For example, when building an application with the |nRFVSC|, you need to add the following parameters in the **Extra CMake arguments** field on the **Add Build Configuration view**: ``-DFP_MODEL_ID=0xFFFFFF -DFP_ANTI_SPOOFING_KEY=AbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbAbA=``.
+.. note::
+   The Fast Pair partition is only recognized by the sysbuild code when it is defined in the main project domain.
```

Comment:
Isn't this part specific to the Partition Manager?

Reply:
The are small nuances here and there. I can move it to dedicated sections.

https://github.com/nrfconnect/sdk-nrf/blob/main/cmake/sysbuild/fast_pair/hex.cmake#L89

```
  sysbuild_dt_nodelabel(
    bt_fast_pair_partition_node_full_path
    IMAGE
    ${DEFAULT_IMAGE}
    NODELABEL
    "${fp_partition_name}"
    )
```

Reply:
I am not sure if we should go into more details here. Maybe let's leave it as is, but change `project` to `application` (as we refer to `DEFAULT_IMAGE`)?